pkg/actors/actors.go

```diff
@@ -153,6 +165,10 @@ type ActorsOpts struct {
 	TracingSpec      configuration.TracingSpec
 	Resiliency       resiliency.Provider
 	StateStoreName   string
+	InternalActors   map[string]InternalActor
+
+	// MockPlacement is a placement service implementation used for testing
+	MockPlacement PlacementService
```

Comment:
I would avoid to name it as a `Mock...` as this should be tied only for testing purposes. Why not just `PlacementService` ?
```suggestion
	PlacementService PlacementService
```

Reply:
I named it this way because it's only intended to be used for unit testing. I didn't want to give the impression that this is the expected initialization path in production.

Reply:
Yes, I noticed that. My point is: It's being used on tests now but can be used for a different purpose later. That's why I think you should name as it is - which is good actually, meaning that we can provide a custom placement service when creating actors. The usage for tests is only one of possible, this is a way to not "change your code to be testable".

However this is a nit. You can keep it if you want.

Reply:
My thinking is that we should rename it only when we’re ready to support this abstraction beyond testing. I worry that it would be more harmful to rename it now in anticipation of future use cases when we haven’t yet done the work to support those use cases.